package.json

```diff
@@ -49,6 +49,7 @@
     "copy-webpack-plugin": "^6.0.1",
     "css-loader": "^3.0.0",
     "eslint": "^7.1.0",
+    "eslint-plugin-jsdoc": "^32.3.0",
```

Comment:
Even after adding this package, I don't get an error when I run `npm run lint` even though I have removed all docstrings from my local.

Reply:
@algomaster99 
This package does not enforce js-doc to be present.
I found that we could go with the package `eslint-plugin-require-jsdoc`.
However, this rule was deprecated as mentioned [here](https://eslint.org/docs/rules/require-jsdoc).
So should we go with this one? 🤔

Reply:
> This package does not enforce js-doc to be present.

Why have you added it then? Can you tell its purpose? Does it only check the syntax of the JSDoc?

Reply:
@algomaster99 
The purpose of the above-added package was to enforce valid js-docs. It ensures that JSDoc follows certain rules. More details [here](https://www.npmjs.com/package/eslint-plugin-jsdoc). To enforce that JSDoc is present `eslint-plugin-require-jsdoc` should be installed (deprecated) as I mentioned [here](https://github.com/accordproject/cicero-word-add-in/pull/38#discussion_r606952209).

Reply:
@algomaster99 
I noticed a thing that if a function is written then a warning is displayed for no JSDoc comment found.
However, for the arrow function, we do not get any warning as of now.